### PR TITLE
Created armstrong.rb

### PR DESCRIPTION
**Pull request by AnkDos**

### CODE REVIEW

File: code/mathematical-algorithms/armstrong_numbers/armstrong.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ reminder=0
+ sum=0
+ hold=num
+ while(quotient>0)
+   quotient=num/10
+   reminder=num%10
+   sum=sum+cube(reminder)
+   num=quotient
+ end
+if hold==sum
```
Comment (arnavb) [inline code/mathematical-algorithms/armstrong_numbers/armstrong.rb]: return hold==sum also works here.

File: code/mathematical-algorithms/armstrong_numbers/armstrong.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#created by AnkDos
+#You can Simply use '**' to cube ,ex. 3**3=27 ,so cube(remainder) can be written as (reminder**3) 
+def cube(num)
+    return num*num*num
```
Comment (arnavb) [inline code/mathematical-algorithms/armstrong_numbers/armstrong.rb]: no need for the return.

Comment (AnkDos) [reply code/mathematical-algorithms/armstrong_numbers/armstrong.rb]: Yeah mate, But thats just a habbit.
Thanks!

File: code/mathematical-algorithms/armstrong_numbers/armstrong.rb (hunk 1 above)
Comment (AnkDos) [reply code/mathematical-algorithms/armstrong_numbers/armstrong.rb]: yes , But a rubyist will get the trick.Like using '<<' in place of 'push' and etc etc.
I tried making it for the begginers like me.As you can see there is also a useless cube() method too.
Thanks !

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**